boards/cargo/tock_flags.toml

```diff
@@ -0,0 +1,37 @@
+# Licensed under the Apache License, Version 2.0 or the MIT License.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+# Copyright Tock Contributors 2024.
+
+[target.'cfg(all())']
+rustflags = ["-D", "warnings"]
```

Comment:
This unfortunately also promotes warnings to errors in local `cargo build` and `cargo check` -- presumably we don't want this?

Also, the `cfg(all())` is a particularly arcane bit of syntax to indicate that a flag should be added unconditionally, so a small comment could be great.

Reply:
This shouldn't be here. Reverting and fixing elsewhere